pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3167,7 +3207,7 @@
         "4.1.1.0": "4.1.1",
         "4.1.2.0": "4.3.0",
         "4.1.3.0": "4.4.0",
-        "4.1.4.0": "4.5.0",
```

Comment:
**Question:** Is trailing commas not supported? And if so, what kind of issues could this have caused previously?

Reply:
Commas are supported, and this wasn’t an issue. I wrote a tool that generated these changes for me using an existing PackageIndex object model (which I’m pretty sure it uses Newtonsoft.json ) and this tool removed the coma if there was no additional item.